01-coordinated-multipoint.ipynb

```diff
@@ -789,28 +789,28 @@
     "Ocean software selects reasonable default parameters when submitting problems to the \n",
     "quantum computer. However, when performance in crucial, as in these problems, where \n",
     "differences of a percent matter, it is important to configure the QPU with optimal \n",
-    "parameter values. D-Wave's [system documentation](https://docs.dwavesys.com/docs/latest/doc_handbook.html)\n",
+    "parameter values. D-Wave's [system documentation](https://docs.dwavequantum.com/en/latest/quantum_research/index_using.html)\n",
     "provides guidance on optimizing QPU use.\n",
     "\n",
     "Here, two non-default parameters are used:\n",
     "\n",
-    "* [annealing_time](https://docs.dwavesys.com/docs/latest/c_solver_parameters.html#annealing-time) sets \n",
+    "* [annealing_time](https://docs.dwavequantum.com/en/latest/quantum_research/solver_parameters.html) sets \n",
     "    the duration, in microseconds, of quantum annealing time, per read. \n",
     "\n",
     "    For time-critical problems, rather than use the default 20 microseconds with a large \n",
     "    number of reads, a good rule of thumb is to balance the programming and sampling times \n",
-    "    in the overall [QPU access time](https://docs.dwavesys.com/docs/latest/c_qpu_timing.html#qpu-access-time).\n",
+    "    in the overall [QPU access time](https://docs.dwavequantum.com/en/latest/quantum_research/operation_timing.html).\n",
     "    The QPU access time \n",
-    "    [estimation](https://docs.dwavesys.com/docs/latest/c_qpu_timing.html#estimating-access-time) algorithm \n",
+    "    [estimation](https://docs.dwavequantum.com/en/latest/quantum_research/operation_timing.html) algorithm \n",
     "    and companion Ocean tool produce an optimal annealing time of just over 200 microseconds for \n",
     "    most the problems in this example.\n",
     "     \n",
-    "* [Chain strength](https://docs.ocean.dwavesys.com/en/stable/docs_system/reference/generated/dwave.system.composites.FixedEmbeddingComposite.sample.html)\n",
+    "* [Chain strength](https://docs.dwavequantum.com/en/latest/ocean/api_ref_system/generated/dwave.system.composites.FixedEmbeddingComposite.sample.html)\n",
```

Comment:
Is this link correct? [This](https://docs.dwavequantum.com/en/latest/quantum_research/example_and.html#chain-strength) or [this](https://docs.dwavequantum.com/en/latest/quantum_research/example_and.html#chain-strength) would probably make more sense, no?

Reply:
Unresolved comment.

Reply:
Switched to [this](https://docs.dwavequantum.com/en/latest/quantum_research/embedding_intro.html#chain-strength) link, must have accidently repasted a previous copy instead of the correct target. Thanks for catching it, @thisac.